README.md

```diff
@@ -126,3 +126,4 @@ This project is [MIT](./LICENSE) licensed.
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
```

Comment:
- [ ] [OPTIONAL] It would be nice if you remove this `guideline` 👍🏼 

![image](https://user-images.githubusercontent.com/63915024/212677741-7275effd-f788-4799-b9f2-e54a1d78d351.png)